express/parsers/formats/jsonfile.py

```diff
@@ -0,0 +1,21 @@
+import json
+
+class BaseJSONParser(object):
+    """
+    Base JSON parser class.
+
+    Args:
+        path (str): path to the JSON file.
+    """
+
+    def __init__(self, path):
+        self.path = path
+        self.doc = None
+        if self.path and os.path.exists(self.path):
+            with open(self.path, mode='rb') as file:
+                self.doc = self.loads(file.read().decode('utf-8'))
+
+    @staticmethod
+    def loads(text):
+        """Loads JSON from text."""
```

Comment:
Nit: should we just use `json.loads` directly?

Reply:
I agree, but I tried to follow the format of having a `JSON` format explicitly defined within the package structure as requested. We also need to call this function directly here: 
https://github.com/Exabyte-io/express/blob/a8a9cd27433a52c6c764f4f61286910fc1924d55/express/parsers/apps/aiida/formats/zip.py#L24-L25
Please keep in mind that there is no "JSON" file on disk in this case (i.e. there is no `path`), but we are parsing JSON directly from the in-memory stream decoded from the ZIP-file.

Reply:
From discussion: Store static template.

Consider: creating object from esse schema